frontend/src/wrapper/EntrySearch.tsx

```diff
@@ -53,28 +55,34 @@ export default function EntrySearch() {
   const modelList = useMemo(
     () =>
       entries.map((entry) => (
-        <Box key={entry.id} sx={{ maxWidth: '300px' }}>
+        <Box key={entry.id} sx={{ maxWidth: '400px' }}>
           <Link href={`/${entry.kind}/${entry.id}`} noLinkStyle>
-            <ListItemButton>
-              <ListItemText
-                primary={entry.name}
-                secondary={entry.description}
-                slotProps={{
-                  primary: {
-                    style: {
-                      whiteSpace: 'nowrap',
-                      textOverflow: 'ellipsis',
-                      overflow: 'hidden',
-                      color: theme.palette.primary.main,
+            <ListItem
+              disablePadding
+              secondaryAction={<Chip label={toTitleCase(EntryKindLabel[entry.kind])} size='small' />}
+            >
+              <ListItemButton>
+                <ListItemText
+                  primary={entry.name}
+                  secondary={entry.description}
+                  slotProps={{
+                    primary: {
+                      style: {
+                        whiteSpace: 'nowrap',
+                        textOverflow: 'ellipsis',
+                        overflow: 'hidden',
+                        color: theme.palette.primary.main,
+                        maxWidth: '400px',
```

Comment:
Because this is set to the same width as the surrounding box, really long names will appear under the entry kind chip. I'd suggest instead of settings this to a specific amount of pixels, giving it a `marginRight`. If you change `style` to `sx` on line 70 you can use `mr: 6`, which should give enough padding.